hw/ip/otbn/dv/verilator/otbn_top_sim_waivers.vlt

```diff
@@ -8,3 +8,13 @@ lint_off -rule WIDTH -file "*/otbn_top_sim.sv" -match "*'OTBN_IMEM_SIZE' generat
 lint_off -rule WIDTH -file "*/otbn_top_sim.sv" -match "*'OTBN_DMEM_SIZE' generates 22 bits*"
 lint_off -rule WIDTH -file "*/otbn_top_sim.sv" -match "*'ImemStartAddr' generates 32 bits*"
 lint_off -rule WIDTH -file "*/otbn_top_sim.sv" -match "*'stack_wr_ptr_q' generates 4 bits*"
+
+// TODO: Tracer specific lint file?
+lint_off -rule UNUSED -file "*/otbn_core_tracer.sv" -match "*Bits of signal are not used: 'ti'*"
+lint_off -rule BLKSEQ -file "*/otbn_core_tracer.sv" -match "*Blocking assignments (=) in sequential (flop or latch) block*"
+
+// Flag ISPR (3) has it's own trace signals due to special handling
```

Comment:
Nit: This should be "its"

Reply:
Thanks for fixing the other copy of this code, but I think this file still has the problem.

Reply:
Ah that's because it shouldn't be here in the first place (note the TODO: note above a reminder to myself it should become a separate lint file which I did but failed to remove it from here).